website/assets/js/myscript.js

```diff
@@ -91,10 +89,7 @@ function myFunction() {
 }
 
 function uploadImage() {
-  const myDiv = document.getElementById("imageContainershot");
-  const images = myDiv.getElementsByTagName("img");
-  while (images.length > 0) {
-    images[0].parentNode.removeChild(images[0]);
-  }
   alert("Your entry has been submitted successfully!");
 }
+
```

Comment:
LGTM

Reply:
This could be converted from an alert to modal as we have in other flows, just an enhancement.